dallinger/command_line.py

```diff
@@ -414,16 +414,15 @@ def deploy_sandbox_shared_setup(verbose=True, app=None, web_procs=1, exp_config=
     ]
     for cmd in cmds:
         subprocess.check_call(
-            cmd + " --app " + app_name(id), stdout=out, shell=True)
+            cmd.split(" ") + ["--app", app_name(id)], stdout=out)
```

Comment:
I know it's annoying to have to change all the commands, but I'd be wary of using `split` as this could cause bugs if you have any quoted spaces in the above commands.